openedx_external_enrollments/models.py

```diff
@@ -101,3 +101,17 @@ class Meta:
         Model meta class.
         """
         app_label = "openedx_external_enrollments"
+
+
+class PathstreamEnrollment(ExternalEnrollment):
```

Comment:
@Jacatove, I think we don't need an extra model to store Pathstream enrollments, you can reuse ExternalEnrollments model, and add is_uploaded field inside meta field. Does it make sense?

Reply:
@ivanvgh It makes sense but there is a hidden issue, filtering JSONField is not supported with our setup. It needs MySQL5.7+.

FYI: @diegomillan @Squirrel18

Reply:
However, As discussed with @ivanvgh,  It will use `icontains` to filter.

Reply:
Yes, that could be a possible solution, not fancy but it avoids creating an extra model or even an extra field on `ExternalEnrollment`(field that is very specific to Pathstream which should go inside meta as an extra field), once we migrate to the next version in the short term, this filter should be refactored as e.g. `ExternalEnrollment.objects.filter(meta__is_uploaded=False)` .